src/codelta_backend/src/team.rs

```diff
@@ -0,0 +1,85 @@
+use icrc_ledger_types::icrc1::account::DEFAULT_SUBACCOUNT;
+
+use crate::{
+    member::{Member, MEMBERS},
+    topic::Topic,
+};
+
+pub struct Team<'a> {
+    /// Account derived from canister - canister controlled
+    pub sub_account: &'a [u8; 32],
+
+    pub topic: Topic,
+
+    pub members: &'a [&'a Member<'a>],
+}
+
+/// Teams are split by topic which they review
+pub const TEAMS: &[Team] = &[
```

Comment:
This is nicely organised. I think a minor issue is that at the moment this maps subaccounts to NNS topics 1-to-1, but there are some NNS topics that need to map to the same subaccount.

I think API Boundary Node Management needs to be treated as the same 'topic' as Subnet Management. It might be worth referring to these as 'Allocation' instead of topic to avoid that confusion.

Participant Management also needs to be part of the same allocation as Node Admin (at least according to the current funding setup).